src/BudgetManager/BudgetManager.API/Controllers/TransactionController.cs

```diff
@@ -28,6 +28,18 @@ public async Task<IActionResult> GetTransactionById(Guid id, CancellationToken c
             return response == null ? NotFound() : Ok(response);
         }
 
+        [HttpGet("page/{pageNumber:int}")]
+        public async Task<IActionResult> GetTransactionPage(int pageNumber, CancellationToken cancellationToken)
+        {
+            if (pageNumber <= 0)
+            {
+                return BadRequest();
+            }
+
+            var pageResult = await _mediator.Send(new GetTransactionsByPageQuery(pageNumber), cancellationToken);
+            return Ok(pageResult);
```

Comment:
it's not a good aproach of pagination. FE side should know information about pages( we need to ask them about it. What king of info does they want to receive from our side), and we need to send them response with info about pages and  transactions for this page.

Reply:
Here some links of examples ( sorry, they are in russian, but examples very easy to understand)
1) https://metanit.com/sharp/mvc5/5.13.php   
2) https://metanit.com/sharp/aspnet5/12.7.php

Reply:
The pagination example in the links above is more relevant to MVC projects. How can we apply it to our case?

Reply:
Just create page class with fields that is needed for FE side (we need to ask them what info does thay want to receive from our side) and then send them info about current page with data from db for this page
as this acticles describe

Reply:
Because in the case that you made they can not receive any info about pages

Reply:
Okay, I think we should wait until we have clear requirements from FE side. I'll close the PR for now then.